【BugFix】completion接口echo回显支持

AuferGachet opened the pull request:
本次支持了completion 接口的单/多prompt的echo回显：
*  非流式上逻辑上支持，但最新版本报错，已修复。
*  流式上最新版本不支持， 已做相应的支持。
具体而言，针对于v1/completion接口
1. 非流式
请求：
`curl -X POST http://127.0.0.1:8566/v1/completions \
  -H "Content-Type: application/json" \
  -d '{
    "prompt": "以下是你的自我介绍：",
    "echo": true
  }' `
结果：（能够做到prompt拼接在回复文本之前）
..."text":"以下是你的自我介绍：\n这是我的第一篇博客，欢迎大家来阅读。\n我是来自2018级计算机科学与技术专业的刘文博。"...
2. 流式
实现相同功能，能够做到prompt拼接在第一包返回的文本之前。

AuferGachet committed:
wenxin-tools-511,修复v1/completion无法回显的问题。

AuferGachet committed:
支持多prompt的回显

AuferGachet committed:
支持多prompt情况下的流式回显

AuferGachet committed:
补充了 completion 接口支持 echo 的单元测试

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/FastDeploy?pullRequest=3245) <br/>All committers have signed the CLA.

AuferGachet committed:
Merge branch 'PaddlePaddle:develop' into wenxin-tools-551

AuferGachet committed:
pre-commit

AuferGachet committed:
移除了多余的test文件

AuferGachet committed:
修复了completion接口echo支持的单测方法

AuferGachet committed:
补充了单元测试文件

AuferGachet committed:
补充单测

AuferGachet committed:
unittest

AuferGachet committed:
补充单测

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
修复单测

AuferGachet committed:
删除不必要的assert.

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
重新提交

AuferGachet committed:
重新提交
Merge branch 'wenxin-tools-551' of https://github.com/AuferGachet/FastDeploy into wenxin-tools-551

AuferGachet committed:
更新测试方法

AuferGachet committed:
ut

AuferGachet committed:
验证是否是正确思路单测

AuferGachet committed:
验证是否是正确思路单测

AuferGachet committed:
验证是否是正确思路单测3

AuferGachet committed:
优化单测代码，有针对性地缩小单测范围。

AuferGachet committed:
优化单测代码2，有针对性地缩小单测范围。

AuferGachet committed:
优化单测代码3，有针对性地缩小单测范围。

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
support 'echo' in chat/completion.

AuferGachet committed:
删除无用注释。
Merge branch 'wenxin-tools-551' of https://github.com/AuferGachet/FastDeploy into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
update

AuferGachet committed:
update

AuferGachet committed:
update

AuferGachet committed:
update

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
update

AuferGachet committed:
update
Merge branch 'wenxin-tools-551' of https://github.com/AuferGachet/FastDeploy into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet commented:
retry

AuferGachet commented:
retry

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
update

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
补充了关于tokenid的单元测试

AuferGachet committed:
update

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
修正index错误

AuferGachet committed:
修复`
Merge branch 'wenxin-tools-551' of https://github.com/AuferGachet/FastDeploy into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551

AuferGachet committed:
修正index错误

AuferGachet committed:
update
Merge branch 'wenxin-tools-551' of https://github.com/AuferGachet/FastDeploy into wenxin-tools-551

AuferGachet committed:
Merge branch 'develop' into wenxin-tools-551
